schema/lc.proto

```diff
@@ -177,6 +177,7 @@ message VCNSearchRequest {
   string attachName = 10;
   string artifactName = 11;
   string UID = 12;
+  bytes seekKey = 13;
```

Comment:
Please call it like `continuationToken` - let's make it totally opaque to VCN - it does not need to know that this is immudb's search key.

That would also require that `continuationToken` to be returned in `EntriesList` response - if empty then it means that there are no more entries. Otherwise VCN should just pass the `continuationToken` value from previous response to the next request.

Reply:
@byo VCN [already knows](https://github.com/codenotary/vcn-enterprise/blob/master/pkg/api/lc_artifact.go#L533-L542) what a `Key` is.

Reply:
but ok. I changed